data/osama-22.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Osama Malik",
+  "type": "personal",
+  "bio": "Full Stack Developer. Love travel, meet new people and become a part of their story.",
+  "links": [
+    {
+      "name": "Follow me on Linkedin",
+      "url": "www.linkedin.com/in/osama-malik-15a68018b",
```

Please use the `https://` protocol here, else it sometimes might consider it as a sub-directory in database and use the host domain. 

```suggestion
      "url": "https://www.linkedin.com/in/osama-malik-15a68018b",
```